src/main/java/liquibase/ext/databricks/snapshot/jvm/TableSnapshotGeneratorDatabricks.java

```diff
@@ -57,5 +63,10 @@ protected DatabaseObject snapshotObject(DatabaseObject example, DatabaseSnapshot
         }
         return table;
     }
+    //TODO another way of getting Location is query like
```

Comment:
Do we still need this TODO?

Reply:
Yes, we may have to make additional calls in the scope of other snapshot tickets. Let's remove this in scope of DAT-18853